apps/aecontract/src/aect_contracts.erl

```diff
@@ -118,6 +135,7 @@ serialize(#contract{} = C) ->
       [ {pubkey, pubkey(C)}
       , {balance, balance(C)}
       , {height, height(C)}
+      , {nonce, nonce(C)}
```

Comment:
Chain format breaking change hence needed:
* Block protocol version bump;
* Genesis block version bump (as no support for hard fork without restarting from genesis yet;
* Deploy DB path bump.

Reply:
.... and serialization doc at https://github.com/aeternity/protocol/blob/28d6eae0ec273a7a6c9a1c07272ff31ad12765b4/serializations.md#contract

Reply:
(Following comment is optional for this PR.) How can existing contract in state tree in chain be converted to this new format? Nonce 0?

Reply:
Serialization doc updated in protocol branch PT-154173362-vm-chain-api